assets/css/structure.css

```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: var(--primary-font);
+  background-color: var(--primary-color);
+  place-items: center;
```

Comment:
what is this doing? i think it's extra and you can remove it.

Reply:
The total body?what about background color?